csrc/utils.h

```diff
@@ -28,6 +30,13 @@ bool is_zero_sized_tensor(const std::shared_ptr<c10::TensorType>& tensor_type);
 bool is_cpu_scalar(const at::Tensor& tensor);
 bool is_cpu_scalar(const c10::TensorType& tensor_type);
 
+// query codegen output arch and target
```

Comment:
```suggestion
// Query the target GPU version number NVRTC compiles generated CUDA kernels for
```

Also, the function name doesn't seem to be representing well what it actually does. Maybe something like `queryTargetGPUVersion`?

Does this need to be here? Isn't it only used by executor.cpp?

Reply:
That's a good point.

This file was shared among multiple backends previously (NNC and legacy fuser). But since that file is not shipped with binary, I need to move it here. I'll dump this inside `executor.cpp` then.